antea/reco/petit_reco_functions.py

```diff
@@ -148,3 +148,60 @@ def compute_charge_ratio_in_corona(df: pd.DataFrame,
     tot_ch_d = df[df.tofpet_id==tofpet_id]  .groupby(evt_groupby)[variable].sum()
     cor_ch   = df[df.sensor_id.isin(corona)].groupby(evt_groupby)[variable].sum()
     return (cor_ch/tot_ch_d).fillna(0)
+    
+    
+def apply_same_sensor_id_in_groups_of_4sensors(df): 
+    
+    """
```

Comment:
In the description I would specify that this function is to be used with the FBK sensors numbered in a specific way (100... 163, 200... 263 etc.).